conf/zeppelin-site.xml.template

```diff
@@ -219,5 +219,11 @@
   <description>Anonymous user allowed by default</description>
 </property>
 
+<property>
+  <name>zeppelin.websocket.max.text.message.size</name>
+  <value>1024000</value>
+  <description>Size in characters of the maximum text message to be received by websocket. Defaults to 1024000</description>
```

Comment:
@felixcheung that message is taken from http://archive.eclipse.org/jetty/8.0.0.RC0/apidocs/org/eclipse/jetty/websocket/WebSocket.Connection.html